src/hosted-form/iframe-content/hosted-input-validator.ts

```diff
@@ -0,0 +1,77 @@
+import { cvv, expirationDate, number } from 'card-validator';
+import { pick } from 'lodash';
+import { object, string, StringSchema, ValidationError } from 'yup';
+
+import HostedInputValidateErrorData from './hosted-input-validate-error-data';
+import HostedInputValues from './hosted-input-values';
+
+interface HostedInputValidateOptions {
+    isCardCodeRequired?: boolean;
+}
+
+export default class HostedInputValidator {
+    async validate(values: HostedInputValues, options?: HostedInputValidateOptions): Promise<HostedInputValidateErrorData[]> {
+        const config: { [key in keyof HostedInputValues]: StringSchema } = {
```

Comment:
maybe name it something like `inputSchemas` or `fieldSchemas`

Reply:
I've renamed the variable to `schemas` for brevity.